components/tools/OmeroJava/test/integration/AbstractServerImportTest.java

```diff
@@ -70,7 +71,7 @@ protected ImportLocation importFileset(List<String> srcPaths) throws Exception {
      * @throws Exception
      *             unexpected
      */
-    protected ImportLocation importFileset(List<String> srcPaths, int numberToUpload) throws Exception {
+    protected ImportLocation importFileset(List<String> srcPaths, int numberToUpload, IObject targetObject) throws Exception {
```

Comment:
add a `@param` for this -- there's a setting in Eclipse that will warn of such missing stuff